ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.12__arm64-nightly.yaml

```diff
@@ -183,6 +183,23 @@ tests:
     test:
     - chain: openshift-e2e-test-qe
     workflow: cucushift-installer-rehearse-aws-ipi-sdn
+- as: e2e-aws-ipi-disconnected-private-p2
+  cron: 0 16 */2 * *
```

Comment:
p2 with every 2 days ?
Refer to https://github.com/openshift/release/blob/master/ci-operator/config/openshift/openshift-tests-private/README.md#job-frequency

Reply:
thanks, will change to cron: `0 16 */7 * *`